links/src/iosMain/kotlin/com/splendo/kaluga/links/manager/LinksManager.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ Copyright 2020 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package com.splendo.kaluga.links.manager
+
+import com.splendo.kaluga.links.Links
+import platform.Foundation.NSURL
+
+actual class LinksManager(
+    override val onLinksStateChange: LinksStateChange
+) : BaseLinksManager(onLinksStateChange) {
+
+    override fun handleOutgoingLink(url: String) {
+        val _url = NSURL.URLWithString(url)
```

Comment:
I'd extract link validator as separate interface (or maybe even inject into links manager) to keep it more clean.

Reply:
What do you mean?

Reply:
I mean to have `LinkValidator` interface with method `validate(url: String): Bool` and use it here: `validator.validate(url)` to extract the logic

Reply:
bdd4bdb 04f92b3 1dcf9da 4e4aaa4 7334b2f 577e512